website/blog/2020-01-10-cobalt.md

```diff
@@ -0,0 +1,718 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.0 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+After a long while of silence and with loads of new features we are excited to
+announce the release of Metals v0.8.0 codename `Cobalt`. This release includes a
+huge number of new features and improvements.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+along with contributors from the community.
+
+In this release we merged 201 PRs and closed 24 issues, full details:
+https://github.com/scalameta/metals/milestone/9?closed=1
+
+## TL;DR
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+- goto implementation
+- rename symbol
+- support for Pants build tool
+- run and debug for tests and main methods
+- breakpoints
+- improvements for Bloop integration
+- first code action - import missing symbol
+- worksheet support
+- Add completion to add all abstract members To mention
+
+## Goto implementation
+
+TODO
+
+## Rename symbol
+
+TODO
+
+## Support for Pants build tool
+
+TODO
+
+## Run and debug for tests and main methods
+
+TODO @marek1840 ?
+
+## Breakpoints
+
+TODO @marek1840 ?
+
+## Improvements for Bloop integration
+
+TODO
+
+Restart Bloop automatically if newer version is needed Automatically reconnct
+Switch to to using Bloop Launcher Remove separate plugin for Metals Bloop itself
+resolves semanticdb
+
+## Support for java 11
+
+TODO
+
+Currently tested on CI No warning when using 11 and it will be detected
+automatically
+
+## Worksheet support
+
+TODO
+
+via decoration protocol for coc-metals and vs code via applyEdit and comments +
+hover
+
+## Better support for Vim via coc-metals
+
+TODO @ckipp01 ?
+
+## Import missing symbol
+
+TODO @gabro ?
+
+## Other improvements
+
+- Sort auto-completed exhaustive match'es by declaration order 1174
+- Handle abstract var overrides
+- Add an empty line after the package declaration before import
+- Sort auto-completed exhaustive match'es by declaration order
+- Encode the workspace path directory in build digests
+- Tone down "No SemanticDB" warning
+- Send Tree View update notification on first no-op compilation.
+- Enable goToDefinition when cursor is at end of symbol
+- Prioritize build targets with supported Scala versions.
+
```

Comment:
- formatting on paste for multi-line strings
- json doctor support

Reply:
> formatting on paste for multi-line strings

wasn't it in the previous release?

> json doctor support

It seems not really user facing, we tend to include things that users might use themselves

Reply:
I believe the previous release had `onTypeFormatting`, but didn't support formatting on multiline string pastes. The PR is listed below https://github.com/scalameta/metals/pull/968.

Regarding json doctor, that makes sense, didn't think of that 👍

Reply:
Actually on paste formatting  was included in 0.7.6 - must have forgotten to add the PR there

Reply:
Actually nevermind. The fix was merged later.

Reply:
Added